examples/multiphysics/static-bubble-chns/static-bubble-multiple-folders.py

```diff
@@ -0,0 +1,119 @@
+"""
+Quick user guide: 
+
+This python script is used to plot a slice of the pressure field and the difference between the averaged pressure outside of the bubble and inside the bubble.
+
+How to use this script?
+
+1- Create a first directory (we will call it outputs but feel free to name it as you like).
+2- For each simulation, create a new directory in outputs : output1, output2, output3,...
+3- Run the simulations and store the results in their correct directories
+4- Execute the script with the path of outputs as argument :
+   python3 multiple_folders_new_bubble_detachment_post_processing /PATH/TO/OUTPUTS/DIR 
+5- Enjoy your plots. 
+
+This should be called as follows : python3 static-bubble-multiple-folders.py /PATH/TO/OUTPUTS/DIR
+"""
+# -------------------------------------------
+# Modules
+# -------------------------------------------
+
+from postprocessing_static_bubble import get_pressure_difference, analytical_solution,get_pressure_slice
+import numpy as np
+import os
+import sys
+import scienceplots
+from natsort import os_sorted
+import matplotlib.pyplot as plt
+
+# For plotting nice Latex-style plots and controlling the sizes of the texts on
+# the plot
+plt.style.use(['science', 'ieee'])
+
+SMALL_SIZE = 10
+MEDIUM_SIZE = 10
+BIGGER_SIZE = 15
+plt.rc('font', size=SMALL_SIZE)  # controls default text sizes
+plt.rc('axes', titlesize=SMALL_SIZE)  # fontsize of the axes title
+plt.rc('axes', labelsize=8)  # fontsize of the x and y labels
+plt.rc('xtick', labelsize=SMALL_SIZE)  # fontsize of the tick labels
+plt.rc('ytick', labelsize=SMALL_SIZE)  # fontsize of the tick labels
+plt.rc('legend', fontsize=8)  # legend fontsize
+plt.rc('figure', titlesize=BIGGER_SIZE)  # fontsize of the figure title
+
+# Class containing relevant physical properties of the problem
+class parametres():
+    sigma = 1
+    radius = 0.15
+
+    def set_radius(self, r):
+        self.radius = r
+
+
+prm = parametres()
+
+if (len(sys.argv)==0):
+    raise Exception(f"No outputs folder was specified as an argument of the python script.")
+rootdir = sys.argv[1]
+
+folder_name_list = []
+root, dirs, files = next(os.walk(rootdir, topdown=True))
+
+for dir in dirs:
+    folder_name_list.append(str(root + "/" + dir))
+
+folder_name_list = os_sorted(folder_name_list)
+dirs = os_sorted(dirs)
+
+# Syntax specific to the SciencePlots module 
+with plt.style.context(['science', 'ieee']):
+    fig = plt.figure()
+    ax = fig.add_subplot(111)
+    pparams = dict(xlabel=r'$\text{Rayon} (R) \text{[m]}$',
+                   ylabel=r'$\Delta p \text{[Pa]}$')
+
+plots = []
+
+for i in range(len(folder_name_list)):
+    # Get the radius of the case from the directory name. (Works if the subdirectories were generated using the generate_cases_locally.py script)
```

Comment:
Maybe add a warning if the subdirectories are not generated using this script?

Reply:
Excellent idea